R/supertibble.R

```diff
@@ -0,0 +1,24 @@
+#' @title
+#' Add supertbl S3 class
+#'
+#' @param x an object to class
+#'
+#' @return
+#' The object with `redcaptidier_supertbl` S3 class
+#'
+#' @keywords internal
+#'
+as_supertbl <- function(x) {
+  class(x) <- c("redcap_supertbl", class(x))
+  x
+}
+
+#' @export
```

Comment:
Should we add an `@inherit` here in case anyone does `?REDCapTidieR:::vec_ptype_abbr.redcap_supertbl`